### PR TITLE
Feature sprint09 tijn multiselect download share

### PR DESCRIPTION
**Pull request by tschmits**

### CODE REVIEW

File: app/scripts/components/modelcontrolmenu.js [hunk 1]
```diff
@@ -11,18 +11,58 @@ var exports = (function () {
 
       return {
         collapseShow: true,
+        selectedDownloads: {
```
Comment (SiggyF) [inline app/scripts/components/modelcontrolmenu.js]: A bit specific. Can we generalize this so we can easier include other models later?

File: app/scripts/components/modelcontrolmenu.js [hunk 2]
```diff
@@ -81,13 +117,39 @@ var exports = (function () {
...
+      downloadSelectedModels: function () {
+        store.downloadSelectedModels(this.selectedDownloads);
+      },
+
+      toggle: function(id, event) {
+        event.stopPropagation();
+        this.selectedDownloads[id] = !this.selectedDownloads[id];
+      },
+      preventCheck: function (event) {
+        event.preventDefault();
```
Comment (SiggyF) [inline app/scripts/components/modelcontrolmenu.js]: default and propagation are handled by vue attributes in the click event handler `click.stop`

File: app/templates/model-control-menu.html [hunk 3]
```diff
@@ -41,6 +41,52 @@
         </a>
       </li>
 
+      <li role="separator" class="divider"></li>
+
+      <li :class="{disabled: numSelectedModels === 0  || !someSelectedModelsAreFinished}">
```
Comment (SiggyF) [inline app/templates/model-control-menu.html]: looks a bit repetitive. Can we make all the download options configurable somehow?

Comment (tschmits) [inline app/templates/model-control-menu.html]: Good idea. Done.

**Pull request merged**